### ISSUE TITLE
Flamenco Genre fails to show me a list of 20 Flamenco Singers

### ISSUE THREAD

**Issue by 1artist** [jellyfin/jellyfin#9889]
I am running the latest version of Jellyfin on my Ubuntu Server v22.04 OS. It is installed as a standalone app without using Docker and Portainer. My Music Library contains 1,500 songs and using Mp3Tag I assigned custom metadata tags to all 1,500 songs.  So for example, I have a Genre named Flamenco and to this Genre I have manually assigned 20 different Flamenco singers to this Flamenco Genre by applying a Flamenco Genre metadata tag to each track.  

Here is the problem, when I select the Flamenco Genre my Jellyfin fails to display to me a list of 20 different Flamenco singers within the Flamenco Genre. It only shows me a black screen. However, if I hit the PLAY button all 20 songs play as they were a single loop but yet the Genre area still remains black.  Is it possible to make Jellyfin show the music tracks as a list within my custom Flamenco Genre? Or is Jellyfin hard coded to only show albums inside the Genre section?  

In short, I do not like to collect my music categorized by albums because the end result is you end up with a lot of useless music eating up a lot of hard drive space.  So this is why using Mp3Tag I have assigned my own custom metadata tags to all of my 1,500 songs. Does Jellyfin provide people like me a way to setup music libraries without using ALBUMS as a required metadata tag category that needs to be used?

Interesting enough my PowerAmp music player app on my smartphone takes full advantage of my custom Genre metadata tags. I hope somebody in this community can clarify to me if it is possible to make my Jellyfin use my custom Genre metadata tags in the same way my PowerAmp app does.  Thank you for your time.

**Comment by srcrist** [comment]
Closing this issue as it does not appear to contain an actionable Jellyfin bug. For general questions or discussion, please come and visit our forums or the Matrix/Discord community. People will be happy to answer questions or provide you with tips and tricks.
